sale_invoice_policy/models/res_config_settings.py

```diff
@@ -7,11 +7,6 @@
 class ResConfigSettings(models.TransientModel):
     _inherit = "res.config.settings"
 
-    sale_default_invoice_policy = fields.Selection(
```

Comment:
Why removing this?

Reply:
`default_get` uses this field https://github.com/odoo/odoo/blob/16.0/addons/sale/models/res_config_settings.py#L11